perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -1396,7 +1454,23 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		public List<Attribute> call(ApiCaller ac, Deserializer parms) throws PerunException {
 			if (parms.contains("service")) {
 				if (parms.contains("resource")) {
-					if (parms.contains("member")) {
+					if (parms.contains("member") && parms.contains("group")) {
```

Comment:
Same like in the last comment. Just think about it. Of course this way technically works too.